msteams-platform/tabs/how-to/create-personal-tab.md

```diff
@@ -10,13 +10,13 @@ ms.date: 02/27/2023
 
 # Create a personal tab
 
+Personal tabs, along with personally-scoped bots, are part of personal apps and are scoped to a single user. They can be pinned to the left pane for easy access. You can also [reorder](#reorder-static-personal-tabs) your personal tabs.
```

Comment:
I don't think you can reorder static personal tabs. You might want to remove the last sentence.